### ISSUE TITLE
Impossible to reset passphrase without login out

### ISSUE THREAD

**Issue by Keruspe** [keybase/keybase-issues#1381]
You can recover when you're logged out, but it's impossible when you're not.
It can be a problem when a user creates an account with a temporary password, and wants to create a key several days later having forgotten the password

**Comment by R-Rudolf** [comment]
Yes, I think there should be some explanations at the passphrase input. I don't know if it is possible to change just the passphrase and not the whole keypair. Is it possibe, or just I don't find the tutorial?

**Comment by scottmccool** [comment]
This is particularly problematic with https://github.com/keybase/keybase-issues/issues/1172 -- in my case during signup I generated a passphrase using lastpass and didn't copy it to clipboard.  I went ahead with verification but am now left logged into the site but unable to reset my password.

Pretty minor since logging out does let you recover, but it should be possible to trigger the account recovery email while logged in.

I've since logged out and recovered it that way, but the otherwise smooth signup process was a little bumpy because of the interaction with these two issues.
